packages/apps/src/index.tsx

```diff
@@ -27,7 +26,7 @@ const rootElement = document.getElementById(rootId);
 //  - http://localhost:3000/?rpc=wss://substrate-rpc.parity.io/#/explorer
 //  - http://localhost:3000/#/explorer?rpc=wss://substrate-rpc.parity.io
 const urlOptions = queryString.parse(location.href.split('?')[1]);
-const _wsEndpoint = urlOptions.rpc || process.env.WS_URL || settings.apiUrl;
+const _wsEndpoint = urlOptions.rpc || settings.apiUrl;
```

Comment:
Wouldn't it make more sense to use `process.env.WS_URL` at the end of the list, i. e. 

```ts
const _wsEndpoint = urlOptions.rpc || settings.apiUrl || process.env.WS_URL;	
```